manifests/n/Ngrok/Ngrok/3.1.1/Ngrok.Ngrok.locale.en-US.yaml

```diff
@@ -0,0 +1,32 @@
+# Created with YamlCreate.ps1 v2.2.2 $debug=NVS0.CRLF.5-1-19041-2364.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Ngrok.Ngrok
+PackageVersion: 3.1.1
+PackageLocale: en-US
+Publisher: ngrok, Inc.
+PublisherUrl: https://ngrok.com
+# PublisherSupportUrl:
+PrivacyUrl: https://ngrok.com/privacy
+Author: ngrok, Inc.
+PackageName: Ngrok
+PackageUrl: https://ngrok.com
+License: Proprietary
+# LicenseUrl:
+# Copyright:
+# CopyrightUrl:
+ShortDescription: Tunnel local ports to public URLs and inspect traffic
```

Comment:
```suggestion
ShortDescription: ngrok is a simplified API-first ingress-as-a-service that adds connectivity,security, and observability to your apps with no code changes
```
I would suggest to stick with the description on the PackageUrl https://ngrok.com

Reply:
I used the descriptions returned by `ngrok -h`.
![image](https://user-images.githubusercontent.com/12696894/218344534-5772a02a-e3bf-4b6b-90a3-d2fc574e3cb8.png)

Thoughts?

Reply:
Ah right, I was wondering where you got that description from. Both work for me as they come directly from the publisher.